src/Serilog.Settings.Configuration/Settings/Configuration/ConfigurationReaderOptions.cs

```diff
@@ -56,6 +57,16 @@ public ConfigurationReaderOptions() : this(dependencyContext: null)
     /// </summary>
     public IFormatProvider FormatProvider { get; init; } = CultureInfo.InvariantCulture;
 
+    /// <summary>
+    /// Called when a log level switch is created while reading the configuration.
+    /// Log level switches are created either from the <c>Serilog:LevelSwitches</c> section (declared switches) or the <c>Serilog:MinimumLevel:Override</c> section (minimum level override switches).
+    /// <list type="bullet">
+    ///   <item>For declared switches, the switch name includes the leading <c>$</c> character.</item>
+    ///   <item>For minimum level override switches, the switch name is the (partial) namespace or type name of the override.</item>
+    /// </list>
+    /// </summary>
+    public Action<string, LoggingLevelSwitch> OnAddedLevelSwitch { get; init; }
```

Comment:
`OnLevelSwitchCreated` or `OnLevelSwitchConfigured` perhaps?

Reply:
I went with `OnLevelSwitchCreated` since the documentation says _Called when a log level switch is created_. I force-pushed b483da5de199d764f0dc36fa2418bd086dee0715 on top of the `dev` branch.